src/modules/aws_account@0.0.1/index.ts

```diff
@@ -4,12 +4,10 @@ import { AWS, } from '../../services/gateways/aws'
 import { AwsAccountEntity, } from './entity'
 import * as allEntities from './entity'
 import { Context, Crud, Mapper, Module, } from '../interfaces'
-import { awsAccount1644463885917, } from './migration/1644463885917-aws_account'
+import * as metadata from './module.json'
 
 export const AwsAccount: Module = new Module({
-  name: 'aws_account',
-  version: '0.0.1',
-  dependencies: [],
+  ...metadata,
```

Comment:
This can probably also be automated in the `interfaces.ts` file. Not sure if it makes sense to do something like that now or later.

Reply:
The automation would be just the import of the `metadata`?

Reply:
Yeah. It's not *exactly* being declared twice, but since this now requires a `module.json` in a particular place to generate the migration, why not have the code assume that and not need to import here to re-export to the interface class?